test/report-design-viewer/errors.js

```diff
@@ -0,0 +1,174 @@
+
+module.exports = [
+    [
+        {
+            stepName:          'Check arrays equality',
+            expected:          '"12345678901"',
+            actual:            '"00000000000"',
+            relatedSourceCode: 'eq(["12345678901"], ["00000000000"])',
+            key:               0,
+            isArrays:          true,
+            code:              TYPE.eqAssertion,
+
+            diffType: {
+                isStrings: true,
+                diffIndex: 0
+            }
+        },
+        {
+            stepName:          'Comparison two strings',
+            relatedSourceCode: 'notEq("test", "test")',
+            actual:            '"test"',
+            expected:          '"test"',
+            code:              TYPE.notEqAssertion
+        },
+        {
+            stepName:          'Check value',
+            relatedSourceCode: 'ok(false)',
+            actual:            'false',
+            code:              TYPE.okAssertion
+        },
+
+        {
+            stepName:          'Check value',
+            relatedSourceCode: 'notOk("test")',
+            actual:            '"test"',
+            code:              TYPE.notOkAssertion
+        },
+        {
+            code: TYPE.xhrRequestTimeout
+        },
+        {
+            code: TYPE.iframeLoadingTimeout
+        },
+        {
+            code:     TYPE.inIFrameTargetLoadingTimeout,
+            stepName: 'Do action in iframe'
+        },
+        {
+            code:    TYPE.urlUtilProtocolIsNotSupported,
+            destUrl: 'ftp://host:8080/'
+        },
+        {
+            code:      TYPE.uncaughtJSError,
+            scriptErr: 'Cannot read property \'name\' of null',
+            pageUrl:   'http://host:8080/index.aspx'
+        }
+    ],
+    [
+        {
+            code:      TYPE.uncaughtJSErrorInTestCodeStep,
+            stepName:  'Step',
+            scriptErr: 'error'
+        },
+        {
+            code:     TYPE.storeDomNodeOrJqueryObject,
+            stepName: 'Share Jquery object between steps'
+        },
+        {
+            code:              TYPE.emptyFirstArgument,
+            stepName:          'Click Login button',
+            relatedSourceCode: 'act.click(null, {offsetX: 0, offsetY: 1000})',
+            action:            'act.click'
+        },
+        {
+            code:              TYPE.invisibleActionElement,
+            stepName:          'Drag options panel',
+            relatedSourceCode: 'act.drag($("button.login"), $("div.dock"), {dragOffsetX: 1, dragOffsetY: 2})',
+            action:            'act.drag',
+            element:           '&lt;button class=&quot;login&quot;&gt;&lt;/button&gt;'
+        },
+        {
+            code:              TYPE.incorrectDraggingSecondArgument,
+            stepName:          'Drag options panel',
+            relatedSourceCode: 'act.drag($("button.login"), 1)',
+            action:            'act.drag'
```

Comment:
We don't need `action` for this error, it was accidently addded, but I've removed it in the recent commit.